patroni/dcs/zookeeper.py

```diff
@@ -114,7 +114,7 @@ def set_ttl(self, ttl):
             return True
 
     def set_retry_timeout(self, retry_timeout):
-        self._client._retry.deadline = retry_timeout
+        self._client.retry.deadline = retry_timeout
```

Comment:
Good point.

kazoo 1.3.1 (debian 8):
```
>>> import kazoo.client
>>> c = kazoo.client.KazooClient()
>>> type(c.retry)
<class 'kazoo.retry.KazooRetry'>
>>> type(c._retry)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'KazooClient' object has no attribute '_retry'
```
kazoo 2.2.1 (debian 9):
```
>>> import kazoo.client
>>> c = kazoo.client.KazooClient()
>>> type(c.retry)
<type 'function'>
>>> type(c._retry)
<class 'kazoo.retry.KazooRetry'>
```
I see the following options:
1. catch this `AttributeError` in a try/catch block and set the `retry` object as a fallback
2. use a couple of `if` statements to type check the `_retry` and `retry` objects and set the one with the `kazoo.retry.KazooRetry` type
3. require a minimum version of python-kazoo and ignore this exception

Do you have a preference or a better solution?